tests/components/sensor/test_rfxtrx.py

```diff
@@ -44,12 +44,7 @@ def test_old_config_sensor(self):
         entity = rfxtrx_core.RFX_DEVICES['sensor_0502']['Temperature']
         self.assertEqual('Test', entity.name)
         self.assertEqual(TEMP_CELSIUS, entity.unit_of_measurement)
-        self.assertEqual(14.9, entity.state)
-        self.assertEqual({'Humidity status': 'normal', 'Temperature': 14.9,
-                          'Rssi numeric': 6, 'Humidity': 34,
-                          'Battery numeric': 9,
-                          'Humidity status numeric': 2},
-                         entity.device_state_attributes)
+        self.assertEqual(None, entity.state)
```

Comment:
I think the tests could be improved by reading this data from the state machine. That way you are guaranteed that the data you output is written correctly (and that's what's we want to test after all)